portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,41 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+.center-text {
+  text-align: center;
+  margin-top: 35px;
+  }
+
+.center-image {
+	text-align: center;
```

Comment:
nit: Let's fix the indentation for this property.

Reply:
fixed